tern/__main__.py

```diff
@@ -67,6 +68,14 @@ def create_top_dir():
         os.mkdir(top_dir)
 
 
+def check_docker_image_string(image_str: str):
+    tag_format = r'.+:.+'
+    digest_format = r'.+@.+:.+'
+    if re.match(tag_format, image_str) or re.match(digest_format, image_str):
+        return True
+    return False
+
+
```

Comment:
Good job! Couple of organization nits:
1. This seems to be the accepted format to reference any container image on a registry. Could you change the name of this function to `check_image_string`?
2. Can you move this function to `tern/utils/general.py`?